### PR TITLE
New rules for Noir Linux

### PR DESCRIPTION
**Pull request by mmatongo**
Changes Introduced add:

- Versioning ruleset for git packages
- Version tracking for baselayout package

### CODE REVIEW

File: 100.prefix-suffix.yaml [hunk 1]
```diff
@@ -80,6 +80,8 @@
 
 - { noflag: no_suffix, ruleset: npackd,          namepat: "(.*?)(-?64)", setname: $1, addflavor: "64" }
 
+- { noflag: no_suffix, ruleset: noir,           namepat: "(.*)-git", ver: "git", setname: $1, rolling: true }
```
Comment (AMDmi3) [inline 100.prefix-suffix.yaml:83]: There are no *-git packages in noir. There probably needs to be a 899 rule to mark `git` versions rolling in noir.

**Review by AMDmi3 — CHANGES_REQUESTED** [review]
Please remove merge commits from this.

File: 100.prefix-suffix.yaml (hunk 1 above)
Comment (mmatongo) [reply 100.prefix-suffix.yaml]: Sorry about that, mistook 100 for 899

**Review by AMDmi3 — APPROVED** [review]

**Comment by AMDmi3** [comment]
Merged in 1ea6ef29e34956720aa834a0c947046eabe5c68f

**Comment by mmatongo** [comment]
> Merged in [1ea6ef2](https://github.com/repology/repology-rules/commit/1ea6ef29e34956720aa834a0c947046eabe5c68f), thank you!

Thanks @AMDmi3

**Pull request closed without merging**